crates/db_schema/src/schema.rs

```diff
@@ -1,45 +1,45 @@
 // @generated automatically by Diesel CLI.
 
 pub mod sql_types {
-    #[derive(diesel::query_builder::QueryId, diesel::sql_types::SqlType)]
-    #[diesel(postgres_type(name = "actor_type_enum"))]
-    pub struct ActorTypeEnum;
+  #[derive(diesel::query_builder::QueryId, diesel::sql_types::SqlType)]
```

Comment:
Undo the formatting changes here, just in case. We have some git `.patch` files that diesel uses, and any changes to these could mess it up. At least I've had that problem in the past.